src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
+@import "./blocks/calendar.scss";
```

Comment:
Consider changing the import order, importing the variables first, which are usually defined before other styles.